ODIN_II/SRC/include/parse_making_ast.h

```diff
@@ -8,6 +8,7 @@ void cleanup_parser();
 void cleanup_hard_blocks();
 void init_parser_for_file();
 void clean_up_parser_for_file();
+ast_node_t *cleanup_specify_items(ast_node_t *root);
```

Comment:
is this needed? i don't see it being used throughout your changes. Please remove this function if you are not using it

Reply:
The `cleanup_specify_items` function is used by `specify_item` in order to free the allocated data structures.

Reply:
can you please point out where that is because your changes does not show any usage of it